test/fixtures/3.0/ping/src/sequence.ts

```diff
@@ -0,0 +1,35 @@
+import {inject} from '@loopback/context';
+import {
+  FindRoute,
+  InvokeMethod,
+  ParseParams,
+  Reject,
+  RequestContext,
+  RestBindings,
+  Send,
+  SequenceHandler,
+} from '@loopback/rest';
+
+const SequenceActions = RestBindings.SequenceActions;
+
+export class MySequence implements SequenceHandler {
```

Comment:
AFAICT, this is the vanilla sequence with no modifications made. Let's remove this file and use the default built-in sequence provided by LB4.